From 1df686083e8ca9a4d6f3fb4e9d238b11e6007f5c Mon Sep 17 00:00:00 2001
From: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
Date: Mon, 13 Nov 2017 11:43:11 +0900
Subject: [PATCH 07/23] Revert "arm64: dts: r8a7795-es1: Add MFIS device node"

This reverts commit 462c591430d6 ("arm64: dts: r8a7795-es1: Add
MFIS device node")

Signed-off-by: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
index c29b668..2695eff 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
@@ -882,14 +882,6 @@
 				     "renesas,mfis-lock";
 			reg = <0 0xe6260000 0 0x1000>;
 			#hwlock-cells = <1>;
-
-			mfis_as: mfis-as {
-				compatible = "renesas,mfis-as-r8a7795",
-					     "renesas,mfis-as";
-				interrupts = <GIC_SPI 180 IRQ_TYPE_LEVEL_HIGH>;
-				interrupt-names = "eicr0";
-				renesas,mfis-ch = <0>;
-			};
 		};
 
 		pfc: pfc@e6060000 {
-- 
1.9.1

